baseline-cli/build.gradle.kts

```diff
@@ -3,9 +3,14 @@ plugins {
     alias(libs.plugins.shadow)
 }
 
+repositories {
+    mavenCentral()
+}
```

Comment:
That is not necessary, as it's configured by the `common-conventions` plugin [here](https://github.com/JetBrains/qodana-sarif/blob/main/buildSrc/src/main/kotlin/qodana-sarif.common-conventions.gradle.kts#L16)